templates/al2023/provisioners/install-worker.sh

```diff
@@ -66,6 +66,21 @@ sudo dnf install -y \
   pigz \
   python3-dnf-plugin-versionlock
 
+# we need to handle different kernel packages depending on the namespace
+# associated with the minor version.
+if [[ "$(uname -r)" == 6.12.* ]]; then
+  sudo dnf -y install \
+    kernel6.12-devel \
+    kernel6.12-headers
+else
+  sudo dnf -y install \
+    kernel-devel \
+    kernel-headers
+fi
+
+# versionlock kernel packages so they remain consistent.
+sudo dnf versionlock kernel*
```

Comment:
without the `-$(uname -r)` this should be
```suggestion
sudo dnf versionlock 'kernel*'
```
to ensure it's not falsely expanding to match something on path

Reply:
done 🚀